src/commands/synthetics/utils.ts

```diff
@@ -138,6 +138,10 @@ export const getStrictestExecutionRule = (configRule: ExecutionRule, testRule?:
 }
 
 export const hasResultPassed = (result: Result): boolean => {
+  if (result.unhealthy) {
+    return true
+  }
+
```

Comment:
Shouldn't we return an intermediate value to acknowledge that the CI passed in degraded conditions?